package.json

```diff
@@ -82,9 +83,10 @@
     "morgan": "^1.10.0",
     "nanoid": "^3.3.4",
     "nodemailer": "^6.7.8",
-    "pm2": "^5.2.0",
+    "pm2": "^5.3.0",
+    "request": "^2.88.2",
     "shortid": "^2.2.16",
-    "uuid": "^9.0.0",
+    "uuid": "^9.0.1",
```

Comment:
Since this PR was related to UUID version change, Can you show the error or other message which made you change version of pm2, image-hash, and add request

Reply:
yess sure

Reply:
![Screenshot 2024-01-06 003505](https://github.com/PalisadoesFoundation/talawa-api/assets/108184993/9a456276-6ef8-4541-a1ce-716128a31d3e)

I had upgraded all those packages which are mentioned above
pls check and suggest me if they wants to be upgraded or not